emission/tests/netTests/TestWebserver.py

```diff
@@ -58,7 +59,7 @@ def tearDown(self):
 
     def test404Redirect(self):
         from emission.net.api.bottle import response
-        import emission.net.api.cfc_webapp as enacw
+        importlib.reload(enacw)
```

Comment:
why is this an `importlib.reload` instead of a direct `import`?

Reply:
This reload is needed. Though the imported module isn't scoped, it seems the name is, (I'm not sure about this, but just through use that seems to be the case) so the import of `emission.net.api.cfc_webapp` goes at the top so that it can be used both here and in `testResolveAuthWithEnvVar`.

It needs to be reloaded for this test since the initial import was done before the config was set.